app/views/public/api/v1/portals/categories/show.html.erb

```diff
@@ -6,23 +6,23 @@
 <% end %>
 
 <% if !@is_plain_layout_enabled %>
-<div class="bg-woot-50 pt-16 pb-20" style="background: url(/assets/images/hc/grid.svg) color-mix(in srgb, <%= @portal.color %> 10%, white)">
+<div class="bg-woot-50 dark:bg-woot-900 pt-16 pb-20" style="background: url(/assets/images/hc/grid.svg) color-mix(in srgb, <%= @portal.color %> 10%, <%= @theme == 'dark' ? 'black' : 'white' %>)">
```

Comment:
`color-mix(in srgb, <%= @portal.color %> 10%, <%= @theme == 'dark' ? 'black' : 'white' %>)`

This portion is repeated everywhere. Can we create a helper method for the same.